kickstarter/android-oss
# NT-1815: WebViews on staging now loading

# 📲 What
- Migrated to kotlin the v1 `APIRequestInterceptor` and the web `WebRequestInterceptor`
- on staging the webviews within the app load correctly now


# 🛠 How
- Added auth header if staging request intercepted

# 👀 See

| Before 🐛 |
https://user-images.githubusercontent.com/4083656/112177598-51589280-8bb6-11eb-9e01-b1f80a0a3071.mp4
| After 🦋 |
https://user-images.githubusercontent.com/4083656/112177615-54ec1980-8bb6-11eb-9190-785ed184449d.mp4

| --- | --- |
|  |  |

# 📋 QA
- execute on your environment the script `make secrets`
- Launch the app, load staging, navigate to any project and check the campaign screen

# Story 📖

[NT-1815](https://kickstarter.atlassian.net/browse/NT-1815)
